pymongo/collection.py

```diff
@@ -752,7 +755,7 @@ def replace_one(self, filter, replacement, upsert=False,
     def update_one(self, filter, update, upsert=False,
                    bypass_document_validation=False,
                    collation=None, array_filters=None, hint=None,
-                   session=None):
+                   session=None, let=None):
```

Comment:
For all the helpers that didn't support "let" before, can you add a `versionchanged:: 4.1` section?